homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -164,25 +170,25 @@ def __init__(self, data, description: SensorEntityDescription) -> None:
         self._holiday_attrs: dict[str, str] = {}
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType:
         """Return the state of the sensor."""
         if isinstance(self._state, datetime):
-            return self._state.isoformat()
+            return self._state.isoformat()  # type: ignore[unreachable]
         return self._state
 
-    async def async_update(self):
+    async def async_update(self) -> None:
         """Update the state of the sensor."""
         now = dt_util.now()
         _LOGGER.debug("Now: %s Location: %r", now, self._location)
 
         today = now.date()
```

Comment:
Add a type to avoid the ignore.

Reply:
I don't understand, isn't it typed already?